Assets/Scripts/CreatureController.cs

```diff
@@ -19,9 +23,28 @@ public class CharacterController : MonoBehaviour {
         }
     }
 
-    // Purely for testing purposes
     void OnMouseUp () {
-        TakeDamage (20);
+        isSelected = !isSelected;
+
+        if (isSelected) {
+            // Set this creature as the target
+            combatManager.SetTarget(this);
+
+            ShowTargetingArrow();
+        } else {
+            // Reset the target if clicked again
+            combatManager.SetTarget(null);
+
+            HideTargetingArrow();
+        }
```

Comment:
If you change the ShowTargetingArrow() and HideTargetingArrow() into just one call then we can also change this to:
```suggestion
        
        if (isSelected) {
            // Set this creature as the target
            combatManager.SetTarget(this);
        } else {
            // Reset the target if clicked again
            combatManager.SetTarget(null);
        }
        setTargetingArrow(isSelected);
```

Reply:
Resolved along with above comment.